vrc-get-gui/app/repositories/page.tsx

```diff
@@ -356,25 +348,23 @@ function Confirming(
 		add: () => void,
 	}
 ) {
-	const {t} = useTranslation();
-
 	return (
 		<>
-			<DialogBody>
-				<Typography>{t("name: {{name}}", {name: repo.display_name})}</Typography>
-				<Typography>{t("url: {{url}}", {url: repo.url})}</Typography>
-				<Typography>{t("packages")}</Typography>
-				<List className={"max-h-[50vh] overflow-y-auto"}>
+			<DialogBody className={"max-h-[50vh] overflow-y-auto"}>
+				<Typography>{tc("name: {{name}}", {name: repo.display_name})}</Typography>
+				<Typography>{tc("url: {{url}}", {url: repo.url})}</Typography>
+				<Typography>{tc("packages: ")}</Typography>
```

Comment:
```suggestion
				<Typography>{tc("packages:")}</Typography>
```
こっちにだけ空白が入ってて、恐らくこれが理由でこの部分のlocalesが機能してないです
逆に向こうに空白入れるべきという可能性はあります